andela/troupon
# #117903759 Create new fixtures

### What does this chore do?

This chore introduces new fixtures which can be used to seed the Troupon database.

### Description

The new deals have been streamlined to have clearer information, relevant and consistent details, and remove such mistakes as having deals which have negative discounts.

### How should this be manually tested?

-Load the fixtures by running the command `python troupon/manage.py loaddata initial_data.json`.
-Navigate to `/deals/` on the server running the app to see the data loaded.

### Relevant pivotal tracker stories
 #117903759
